helpers/handlePrizePoolCoreDeploy.ts

```diff
@@ -0,0 +1,55 @@
+import { Signer } from "@ethersproject/abstract-signer";
+import { cyan, green } from "chalk";
+import { deployContract } from './deployContract'
+
+export interface handlePrizePoolCoreDeployConfig {
+  decimals: number | string,
+  yieldSourcePrizePool: string,
+  drawDufferCardinality: number | string,
+  prizeDistributionBufferCardinality: number | string,
+}
+
+export async function handlePrizePoolCoreDeploy(
+  deploy: Function,
+  deployer: string,
+  ethers: any,
+  config: handlePrizePoolCoreDeployConfig = {
+    decimals: 18,
+    yieldSourcePrizePool: '',
+    drawDufferCardinality: "",
+    prizeDistributionBufferCardinality: ""
+  }
```

Comment:
I think named params here would be simpler than creating a separate typescript type.  Less overhead.

Reply:
Personally as a developer I wouldn't want to worry about ordering of contract addresses, and instead rely on named parameters for a more simple setup.

Any other opinions @ddesrosier @PierrickGT @aodhgan?

Reply:
Using **named params** is better.  It's not about ordering.

What you have above is that some of the params are ordered, and the final param is essential a named param.

Having named params for the whole function is a much simpler interface.

Reply:
Here's a [quick explainer](https://blog.jcoglan.com/2021/06/07/named-arguments-in-javascript/)

although I'm guessing you're already familiar with it, maybe just don't know the name